modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/oauth/OAuthConnectionProviderWrapper.java

```diff
@@ -64,12 +66,16 @@ public C connect() throws ConnectionException {
     return super.connect();
   }
 
-  public void updateAuthState() {
-    ResourceOwnerOAuthContext context = getContext();
-
+  private void updateAuthState() {
     final ConnectionProvider<C> delegate = getDelegate();
-    authCodeStateSetter.set(delegate, toAuthorizationCodeState(oauthConfig, context));
+    ResourceOwnerOAuthContext context = getContext();
+    authCodeStateSetter.set(delegate, new UpdatingAuthorizationCodeState(oauthConfig, dancer, context, updatedContext -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement ([squid:S1602](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1602))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7744&issues=AWmiFXYFotIXnmsUruQ3&open=AWmiFXYFotIXnmsUruQ3)

[ISSUE_KEY]: AWmiFXYFotIXnmsUruQ3